database/queries.js

```diff
@@ -0,0 +1,14 @@
+const queries = require('./database/queries');
+const mysqlConfig = require('./config.js');
+
+const getProductInfo= function(id, callback){
+  connection.query('SELECT * FROM products WHERE id = ?', [id], (err, productInfo)=> {
+    if (err) {
```

Comment:
You have the if block but not the else block.  It is bad practice to leave off the else portion of the error handling even if in most cases it works fine.